packages/gatsby-plugin-feed/src/gatsby-ssr.js

```diff
@@ -16,9 +16,15 @@ exports.onRenderBody = ({ setHeadComponents, pathname }, pluginOptions) => {
       if (typeof match === `string`) return new RegExp(match).exec(pathname)
       return true
     })
-    .map(({ output, title }, i) => {
-      if (output.charAt(0) !== `/`) {
-        output = `/` + output
+    .map(({ output, title, link }, i) => {
+      let href = output
+      if(link) {
+        href = link
+      } else {
+        if (output.charAt(0) !== `/`) {
+          href = `/` + output
+        }
+        href = withPrefix(href)
       }
```

Comment:
just as suggestion: maybe with *"little magic"* it can be shorter:

```javascript
let href = link || withPrefix(output.replace(/^\/?/,`/`))
```

Reply:
Hm, that magic does something with a regular expression that did not need a regular expression before.

I can only see the diff part in this comment but looking at that I would prefer the old style tbh. It's better for perf and legibility.

One alternative might be to use `!output.startsWith('/')`, but that'd be a nit.

Reply:
(Nevermind, you can just drop that part altogether)